src/updater.cpp

```diff
@@ -198,7 +198,7 @@ void Updater::setupOnLinux()
     // Setup to automatically download the new release when an update is available
     QObject::connect(feed, &dblsqd::Feed::ready, this, [=]() {
 
-        // only update release builds to prevent auto-update from overwriting your
+        // only update release/public test builds to prevent auto-update from overwriting your
```

Comment:
This comment is not the same as the code. The code says "don't update for development versions", which causes everything else (which may be more than Release/public tests) to update. Maybe we should be more explicit here as well.

Reply:
OK! I think the logic is sound, so I've updated the comment

Reply:
Ok, the approval still stands :shipit: